File: 3110_Score_of_a_String/solution.c
int scoreOfString(char* s) {
    int n = strlen(s), sum = 0;
    for (int i = 0; i < n - 1; i++) {
        int sum_of_ascii = s[i] - s[i + 1];
        sum += abs(sum_of_ascii);
    }
    return sum;
}

File: 3110_Score_of_a_String/solution.py
class Solution:
    def scoreOfString(self, s: str) -> int:
        n = len(s)
        sum_of_ASCII = 0
        for i in range(n - 1):
            sum_of_ASCII += abs(ord(s[i]) - ord(s[i + 1]))

        return sum_of_ASCII